pkg/platform/gitlab.go

```diff
@@ -64,3 +64,8 @@ func (g *GitLab) StoragePrefix(ctx context.Context) (string, error) {
 func (g *GitLab) CommentStatus(ctx context.Context, status Status, params *StatusParams) error {
 	return nil
 }
+
+// CommentEntrypointsSummary reports the summary for the entrypoints command.
```

Comment:
I assume this will be implemented in the future? Do you want to add a todo comment here?

Reply:
Added to top